COMPATIBILITY.md

```diff
@@ -0,0 +1,12 @@
+- [Compatibility with OpenSearch](#compatibility-with-opensearch)
+
+## Compatibility with OpenSearch
+
+The below matrix shows the compatibility of the [`opensearch-project/opensearch-php`](https://packagist.org/packages/opensearch-project/opensearch-php) with versions of [`OpenSearch`](https://opensearch.org/downloads.html#opensearch).
+
+| OpenSearch Version | Client Version |
+|--------------------|----------------|
+| 1.0.x              | 1.0.2          |
+| 1.1.x              | 1.0.2          |
+| 1.2.x              | 1.0.2          |
+| 1.3.x              | 1.0.2          |
```

Comment:
May be expand the versions in the OpenSearch version column like https://github.com/opensearch-project/opensearch-ruby/blob/main/COMPATIBILITY.md to make it consistent?